packages/odyssey-react-mui/src/themes/odyssey/components.tsx

```diff
@@ -1017,6 +1019,84 @@ export const components: ThemeOptions["components"] = {
       }),
     },
   },
+  MuiTooltip: {
+    defaultProps: {
+      arrow: true,
+      enterDelay: 500,
+      enterNextDelay: 250,
+      placement: "top",
+    },
+    styleOverrides: {
+      tooltip: ({ theme, ownerState }) => ({
+        maxWidth: `calc(${theme.mixins.maxWidth} / 2)`,
+        paddingBlock: theme.spacing(2),
+        paddingInline: theme.spacing(3),
+        backgroundColor: theme.palette.grey[900],
+        color: theme.palette.common.white,
+        fontSize: theme.typography.subtitle1.fontSize,
+        lineHeight: theme.typography.subtitle1.lineHeight,
+        ...(ownerState.touch === true && {
+          paddingBlock: theme.spacing(2),
+          paddingInline: theme.spacing(3),
+          fontSize: theme.typography.subtitle1.fontSize,
+          lineHeight: theme.typography.subtitle1.lineHeight,
+          fontWeight: theme.typography.fontWeightRegular,
+        }),
+        [`.${tooltipClasses.popper}[data-popper-placement*="left"] &`]: {
+          transformOrigin: "right center",
+          ...(!ownerState.isRtl
+            ? {
+                marginRight: theme.spacing(3),
+                ...(ownerState.touch === true && {
+                  marginRight: theme.spacing(4),
+                }),
+              }
+            : {
+                marginLeft: theme.spacing(3),
+                ...(ownerState.touch === true && {
+                  marginLeft: theme.spacing(4),
+                }),
+              }),
```

Comment:
I wish there was a cleaner way of doing this where we didn't have to duplicate the entire set of styles.

Reply:
Yeah, this isn't very elegant. I'm writing them like this to adhere as closely as possible to MUI's styles, since this is a theme and not bespoke CSS. ([See Tooltip.js](https://github.com/mui/material-ui/blob/v5.10.5/packages/mui-material/src/Tooltip/Tooltip.js))

This way, we know our selectors match MUI's application 1:1, and it's easy to compare the theme to the source.

Reply:
Sorry, I was typing this comment this morning and got side-tracked.

Can we create a variable and swap the key based on `.isRtl`?

```js
((propName) => ({
  [propName]: theme.spacing(3),
  ...(ownerState.touch === true && {
    [propName]: theme.spacing(4),
  }),
})(
  ownerState.isRtl
  ? 'marginLeft'
  : 'marginRight'
)
```

---

Separately, I prefer positive over negative conditions, so I'd check `isRtl` rather than `!isRtl`.